SparkleSpin/PlayerListViewModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  PlayerListViewModel.swift
+//  SparkleSpin
+//
+//  Created by Britney Smith on 2/27/19.
+//  Copyright © 2019 Britney Smith. All rights reserved.
+//
+
+class PlayerListViewModel {
+    
+    private let playerListModel: PlayerListModel
+    var nameString: String? = nil
+    
+    init(playerListModel: PlayerListModel) {
+        self.playerListModel = playerListModel
+    }
+    
+    func createPlayerWith(name: String) -> PlayerModel {
+        return PlayerModel(name: name)
+    }
```

Comment:
What is the advantage of this function as opposed to just create the object like this `PlayerModel(name: name)`? I think there's probably something you have in mind for the future so I was curious.

Reply:
I wrote this method here because creating a new Player seemed like a model change to me. So it seemed appropriate to place this behavior in the view model.

Reply:
I was thinking about this because your `PlayerModel` is very simple, it used to have the string property only but after thinking about it I think it makes sense, because you will autogenerate the uid.